server/utils/IEBrowserDetection.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+
+/**
+ * @description Check if user is using IE browser and if that's the case show browser unsupported messages
+ *
+ * @returns {boolean}
+ */
+export default function IEBrowserDetection() {
+  const scriptString = `
+    var isIE = document.documentMode;
+    if (isIE) {
+
+      var style = 'background-color: #ffec77; padding: .75rem 1.25rem; margin-bottom: 1rem;' +
+        'border: 1px solid transparent; font-size: 1rem; font-family: "Times"';
+
+      var eng = 'Varaamo doesn’t support Internet Explorer any more. Please use a ' +
+      'newer browser such as <a href="https://www.google.com/chrome/"> Chrome</a>, ' +
+      '<a href="https://www.mozilla.org/en-US/firefox/new/"> Firefox </a> or ' +
+      '<a href="https://www.microsoft.com/en-us/windows/microsoft-edge"> Edge</a>.';
+
+      var fin = 'Varaamo ei tue enää Internet Explorer -selainta. Käytä jotain ' +
+      'uudempaa selainta, kuten <a href="https://www.google.com/chrome/"> Chrome</a>, ' +
+      '<a href="https://www.mozilla.org/en-US/firefox/new/"> Firefox </a> tai ' +
+      '<a href="https://www.microsoft.com/en-us/windows/microsoft-edge"> Edge</a>.';
+
+      var swe = 'Varaamo fungerar inte längre med Internet Explorer. Så var vänlig' +
+      'och använd en nyare  browser som <a href="https://www.google.com/chrome/"> Chrome</a>,' +
+      '<a href="https://www.mozilla.org/en-US/firefox/new/"> Firefox </a> eller' +
+      '<a href="https://www.microsoft.com/en-us/windows/microsoft-edge"> Edge</a>.';
+
+      var eng_tag = document.createElement("p");
```

Comment:
Lines 31-46 could be refactored a bit, e.g:
```
const array = [eng, fin, swe]
array.forEach(function (text, index) {
    var tag = document.createElement("p");
    tag.innerHTML = text;
    tag.setAttribute("style", style);
    document.getElementById("root").appendChild(tag);
});
```
Note: Didn't test this code, but something along those lines should work.